web/src/app/modules/views/controller/modules/view-controller/services/view-controller.service.ts

```diff
@@ -46,6 +48,10 @@ export class ViewControllerService {
         return this.selectedElementService.hasSelection;
     }
 
+    public get historyShown(): boolean {
+        return Type.is(this.navigator.currentElement, CEGModel);
```

Comment:
Do we really provide history only for CEGModels? I think we should include several other classes here, like Process, TestSpecification, and TestProcedure. I know, the branch name is a little differnt to that , but we should do that, because it is just done then.

Reply:
Or, we show the history for all classes (and skip this check by returning true all the time). Are there elements, where we do not want to display the history? @junkerm says this is correct - display it all the time.

Reply:
Ok, will fix.